system-parachains/collectives/collectives-polkadot/src/lib.rs

```diff
@@ -621,6 +639,7 @@ construct_runtime!(
 		Proxy: pallet_proxy::{Pallet, Call, Storage, Event<T>} = 42,
 		Preimage: pallet_preimage::{Pallet, Call, Storage, Event<T>, HoldReason} = 43,
 		Scheduler: pallet_scheduler::{Pallet, Call, Storage, Event<T>} = 44,
+		AssetRate: pallet_asset_rate::{Pallet, Call, Storage, Event<T>} = 45,
```

Comment:
nit: I will prefer to use the default syntax to not explicitly listing things here but it maybe good to keep consistent in this PR and do the change later in one go

Reply:
Yeah. We really need to switch this.